LabBuilder/LabBuilder.psm1

```diff
@@ -347,14 +347,14 @@ function Get-LabConfiguration {
     # Figure out the Config path and load it into the XML object (if we can)
     # This path is used to find any additional configuration files that might
     # be provided with config
-    [String] $ConfigPath = [System.IO.Path]::GetDirectoryName($Path)
+    [String] $ConfigPath = (Get-Location).Path
     [String] $XMLConfigPath = $Configuration.labbuilderconfig.settings.configpath
     if ($XMLConfigPath) {
         if ($XMLConfigPath.Substring(0,1) -eq '.')
         {
             # A relative path was provided in the config path so add the actual path of the
             # XML to it
-            [String] $FullConfigPath = Join-Path -Path $ConfigPath -ChildPath $XMLConfigPath
+            [String] $FullConfigPath = Join-Path -Path $ConfigPath -ChildPath $XMLConfigPath.Substring(2,($XMLConfigPath.Length - 2))
```

Comment:
This code is risky. It may be possible that a path like this is used:
..\mypath\blabla

In which case this code will damage the path.

It may actually be better to just leave it as is - because there although the resulting path might be ugly:
c:\Someplace\.\ConfigFiles\Etc

it will still be valid.

I may be completely misunderstanding though (it won't be the first time :smile:)